nmtwizard/framework.py

```diff
@@ -691,6 +691,8 @@ def _preprocess_file(self, config, input):
             tok_config = config['tokenization']
             src_tokenizer = tokenizer.build_tokenizer(tok_config['source'])
             output = "%s.tok" % input
+            if input.endswith(".gz"):
```

Comment:
Some (internal) frameworks override `_preprocess_file` and `_postprocess_file` and so they will fail to translate gzip files. So unless you want to update them as well, the compression/decompression should be implemented in `trans_wrapper` as you did previously (but still using the `gzip` module).